packages/password-input/src/PasswordInput/PasswordInput.spec.tsx

```diff
@@ -231,49 +231,34 @@ describe('packages/password-input', () => {
         defaultProps.ariaDescribedby,
       );
     });
+  });
 
-    test('input shows error(x) icon', async () => {
-      const { container } = renderPasswordInput({
-        [`aria-describedby`]: defaultProps.ariaDescribedby,
+  describe('without stateNotifications array or aria-describedby', () => {
+    test(`it renders a general error message when stateNotifications is 'error'`, async () => {
+      const { queryByText } = renderPasswordInput({
         stateNotifications: 'error',
         label: defaultProps.label,
+        errorMessage: defaultProps.errorMessage,
```

Comment:
Is there anything preventing a consumer from passing the `successMessage` even if `stateNotification` is `error`? The test name implies that only the error message will render if the state is error.

Reply:
no, a consumer could additionally pass a `successMessage`, and there's always a default set for both. the state will determine which message will render